src/itemStruct.ts

```diff
@@ -81,14 +92,26 @@ export class ItemStructCompletionProvider implements vscode.CompletionItemProvid
 
         this.itemStructNames = (async () => {
             const autocompleteDump = await getAutocompleteDump()
+            const apiDump = await getApiDump()
             return autocompleteDump.ItemStruct.filter(
-                itemStruct => {
-                    return itemStruct.functions.filter(
+                itemStruct => itemStruct.functions.filter(
                         func => func.static,
-                    ).length > 0 || itemStruct.properties.filter((property) => property.static).length > 0
-                },
+                    ).length > 0 || itemStruct.properties.filter((property) => property.static).length > 0,
             ).map(
-                itemStruct => new vscode.CompletionItem(itemStruct.name, vscode.CompletionItemKind.Class),
+                itemStruct => {
+                    const completionItem = new vscode.CompletionItem(itemStruct.name, vscode.CompletionItemKind.Class)
+                    if (itemStruct.name === "Instance") {
+                        const func = itemStruct.functions.find(func => func.name === "new")
+                        if (func !== undefined) {
+                            const [, insertText ] = getFunctionParameters(func, apiDump)
```

Comment:
`const [, insertText ]` ?

Reply:
The first item returned is an Array of parameters, which is not needed so we ignore it as described [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

Reply:
I'd rather you just do `const insertText = getFunctionParameters(func, apiDump)[1]`.